apps/system/js/app_window.js

```diff
@@ -1852,7 +1852,7 @@
           return;
         }
         invoked = true;
-        setTimeout(callback);
+        callback();
```

Comment:
It makes sense to remove this one, however - tryWaitForFullRepaint will always call the function in a timeout already.